float64 left_axis_x
float64 left_axis_y
float64 right_axis_x
float64 right_axis_y

bool button_x
bool button_a
bool button_b
bool button_y

int8 hat_x
int8 hat_y

bool left_bumper
bool right_bumper

bool left_trigger
bool right_trigger

bool button_back
bool button_start

bool left_axis_push
bool right_axis_push
